cassandra-reaper/upgrade-deps.patch

```diff
@@ -1,74 +1,33 @@
-From b75911610c45cf335241565ccac14c384292eea1 Mon Sep 17 00:00:00 2001
-From: Batuhan Apaydin <batuhan.apaydin@chainguard.dev>
-Date: Thu, 11 Jan 2024 01:14:10 +0300
-Subject: [PATCH] upgrade deps
-
-Signed-off-by: Batuhan Apaydin <batuhan.apaydin@chainguard.dev>
----
- 0001-upgrade-deps.patch                 | 26 +++++++++++++++++++++++++
- src/server/pom.xml                      | 14 ++++++-------
- src/server/src/main/resources/shiro.ini | 11 -----------
- 3 files changed, 33 insertions(+), 18 deletions(-)
- create mode 100644 0001-upgrade-deps.patch
-
-diff --git a/0001-upgrade-deps.patch b/0001-upgrade-deps.patch
-new file mode 100644
-index 00000000..0057f857
---- /dev/null
-+++ b/0001-upgrade-deps.patch
-@@ -0,0 +1,26 @@
-+From 088d57487a86e45c0c06c7350b526ddc67927a8d Mon Sep 17 00:00:00 2001
-+From: Batuhan Apaydin <batuhan.apaydin@chainguard.dev>
-+Date: Thu, 11 Jan 2024 01:13:07 +0300
-+Subject: [PATCH] upgrade deps
-+
-+Signed-off-by: Batuhan Apaydin <batuhan.apaydin@chainguard.dev>
-+---
-+ src/server/pom.xml | 2 +-
-+ 1 file changed, 1 insertion(+), 1 deletion(-)
-+
-+diff --git a/src/server/pom.xml b/src/server/pom.xml
-+index 2962017e..06a36bf0 100644
-+--- a/src/server/pom.xml
-++++ b/src/server/pom.xml
-+@@ -29,7 +29,7 @@
-+     <packaging>jar</packaging>
-+ 
-+     <properties>
-+-        <dropwizard.version>2.0.34</dropwizard.version>
-++        <dropwizard.version>2.1.11</dropwizard.version>
-+         <jersey.version>2.33</jersey.version>
-+         <logback.version>1.2.9</logback.version>
-+         <cxf.version>3.4.5</cxf.version>
-+-- 
-+2.39.3 (Apple Git-145)
-+
 diff --git a/src/server/pom.xml b/src/server/pom.xml
-index 2962017e..7421cbc6 100644
+index 58aa96bc..47e4f7b2 100644
 --- a/src/server/pom.xml
 +++ b/src/server/pom.xml
-@@ -29,11 +29,11 @@
-     <packaging>jar</packaging>
- 
-     <properties>
--        <dropwizard.version>2.0.34</dropwizard.version>
-+        <dropwizard.version>2.0.35</dropwizard.version>
```

Comment:
@debasishbsws Why do we downgrade the version of `dropwizard.version` in the pom.xml of v3.6.0 is set to `2.1.12` ?

Reply:
I have removed those lines from the older patch. We are not downgrading here.